nautobot/utilities/testing/__init__.py

```diff
@@ -114,6 +117,16 @@ def setUp(self):
         # Re-populate status choices after database truncation by TransactionTestCase
         populate_status_choices(apps, None)
 
+        # Create the test user and assign permissions
+        self.user = User.objects.create_user(username="testuser")
+        self.add_permissions(*self.user_permissions)
+
+        # Initialize the test client
+        self.client = Client()
+
+        # Force login explicitly with the first-available backend
+        self.client.force_login(self.user)
+
```

Comment:
Any reason we couldn't add a `setUp()` to TestCaseMixin that centralizes this logic as well?

Reply:
Probably. When I was working with Jathan we kept it out, but I will try with the change.